Sources/AccessibilitySnapshot/Core/Swift/Classes/AccessibilityHierarchyParser.swift

```diff
@@ -171,17 +180,20 @@ public final class AccessibilityHierarchyParser {
     /// In most cases, this should use the default value, `UIApplication.shared`.
     public func parseAccessibilityElements(
         in root: UIView,
-        userInterfaceLayoutDirectionProvider: UserInterfaceLayoutDirectionProviding = UIApplication.shared
+        userInterfaceLayoutDirectionProvider: UserInterfaceLayoutDirectionProviding = UIApplication.shared,
+        userInterfaceIdiomProvider: UserInterfaceIdiomProviding = UIDevice.current
```

Comment:
Noting that in the Market snapshot tests we will want to inject a different `UserInterfaceIdiomProviding`, since they're kinda magic: They can run on _any_ 3x iOS device and do swizzling and such to simulate all different device sizes, safe areas, etc, that we need.